cli/pkg/shell/git.go

```diff
@@ -29,6 +31,11 @@ func (c *client) Switch(args ...string) error {
 }
 
 func (c *client) CommitAll(format string, args ...interface{}) error {
+
+	if c.IsPorcelain() {
```

Comment:
For our purposes we don't need the commit to fail if there are no changes. Instead it just warns and continues on.